play-interop-test-java/src/test/scala/akka/grpc/gen/PlayJavaRouterSpec.scala

```diff
@@ -79,21 +73,10 @@ class PlayJavaRouterSpec extends WordSpec with Matchers with BeforeAndAfterAll w
       reply.getMessage shouldBe s"Hello, $name!"
     }
 
-    "allow specifying a different prefix" in {
-      val router = new GreeterServiceRouter(new GreeterServiceImpl(), mat).withPrefix("otherPrefix")
-
-      val uri = Uri(s"http://localhost/otherPrefix/SayHello")
-      router.route(playRequestFor(uri)).isPresent shouldBe true
-
-      val name = "John"
-
-      val handler = router.route(playRequestFor(uri)).get().asInstanceOf[AkkaHttpHandler]
-      val request = akkaHttpRequestFor(uri, HelloRequest.newBuilder().setName(name).build())(HelloRequestSerializer)
-      val response = handler(request).futureValue
-      response.status shouldBe StatusCodes.OK
-
-      val reply = akkaHttpResponse[HelloReply](response).futureValue
-      reply.getMessage shouldBe s"Hello, $name!"
+    "not allow specifying another prefix" in {
```

Comment:
Hm so we drop this feature - that's OK I guess, it was limited and explicitly discouraged from the spec anyway :).